src/lib/utils.ts

```
@@ -191,33 +72,31 @@ export async function uploadFiles(
+					onProgress?.({
+						itemName: file.name,
+						itemIndex: index,
+						progress: progress,
+						total: files.length
+					});
 				}
 			});
 
-			const me = await client.getMe();
-
 			const channelId = user?.channelId!.startsWith('-100')
```

Biome (2.4.4)

[error] Forbidden non-null assertion after optional chaining.

(lint/suspicious/noNonNullAssertedOptionalChain)